client/components/posenet_utils.js

```diff
@@ -26,16 +26,37 @@ function toTuple({y, x}) {
 }
 
 export function drawPoint(ctx, y, x, r, color) {
-  if (x > 20 && x < 170 && y > 10 && y < 110) {
-    console.log('i got hit')
-  }
-
   ctx.beginPath()
   ctx.arc(x, y, r, 0, 2 * Math.PI)
   ctx.fillStyle = color
   ctx.fill()
 }
 
+export function hitAMole(moles, keypoints, minConfidence, ctx) {
+  for (let i = 0; i < keypoints.length; i++) {
+    const keypoint = keypoints[i]
+
+    if (keypoint.score < minConfidence) {
+      continue
+    }
+
+    const {y, x} = keypoint.position
+
+    if (i === 0) {
+      moles.map(mole => {
+        const {top, right, bottom, left} = mole
+        if (x > left && x < right && y > top - 120 && y < bottom - 120) {
+          console.log('mole coords', mole.x, mole.y)
+          console.log('nose coords', x, y)
+
+          //include offset next time
```

Comment:
Currently, coordinates are hard coded. We need to change this to off set the video coordinates.